docker/dockerfiles/Dockerfile_es

```diff
@@ -1,4 +0,0 @@
-FROM elasticsearch:1.7.4
-
-# Install HEAD plugin
-RUN plugin -i mobz/elasticsearch-head
```

Comment:
Does removing this file mean elasticsearch-head is no longer included in our docker containers?